### DISCUSSION TITLE
failed to generate

### DISCUSSION THREAD

**Discussion by isasenoaji** [spatie/browsershot#524] [Q&A] [answered]
Hi, im using linux ubuntu 16.xx. when i tried to save using this library i got this message "For some reason Chrome did not write a file at /tmp/1797499489-0804132001624430435/screenshot.png. Error output: `[0623/134035.886076:ERROR:gpu_init.cc(441)] Passthrough is not supported, GL is swiftshader"

anybody can help me?

**Comment by SudoGetBeer** [comment, marked as answer]
Hm if I think I remember having this error and the installing the dependencies as shown in the README:
sudo apt-get install -y nodejs gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget libgbm-dev

**Comment by condor-bird** [comment]
In addition, the screenshot.png file must be created initially, before writing content to it, since the library does not create the file itself at the specified path.
